vector/src/main/res/layout/view_url_preview.xml

```diff
@@ -7,83 +7,79 @@
     android:layout_height="wrap_content"
     tools:parentTag="androidx.constraintlayout.widget.ConstraintLayout">
 
-    <View
-        android:id="@+id/url_preview_left_border"
-        android:layout_width="2dp"
-        android:layout_height="0dp"
-        android:background="?vctr_content_tertiary"
-        app:layout_constraintBottom_toBottomOf="parent"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toTopOf="parent" />
+    <com.google.android.material.card.MaterialCardView
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        app:cardCornerRadius="8dp">
 
-    <TextView
-        android:id="@+id/url_preview_title"
-        style="@style/Widget.Vector.TextView.Body"
-        android:layout_width="0dp"
-        android:layout_height="wrap_content"
-        android:layout_marginStart="7dp"
-        android:ellipsize="end"
-        android:maxLines="2"
-        android:textColor="?vctr_content_primary"
-        android:textStyle="bold"
-        app:layout_constraintEnd_toStartOf="@+id/url_preview_close"
-        app:layout_constraintStart_toStartOf="@+id/url_preview_left_border"
-        app:layout_constraintTop_toTopOf="parent"
-        tools:text="Jo Malone denounces her former brand's John Boyega decision" />
+        <LinearLayout
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:orientation="vertical">
 
-    <ImageView
-        android:id="@+id/url_preview_image"
-        android:layout_width="0dp"
-        android:layout_height="157dp"
-        android:layout_marginTop="16dp"
-        android:importantForAccessibility="no"
-        android:scaleType="fitStart"
-        app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintStart_toStartOf="@+id/url_preview_title"
-        app:layout_constraintTop_toBottomOf="@+id/url_preview_title"
-        tools:src="@tools:sample/backgrounds/scenic" />
+            <ImageView
+                android:id="@+id/url_preview_image"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:adjustViewBounds="true"
+                android:importantForAccessibility="no"
+                android:scaleType="fitXY"
+                tools:src="@tools:sample/backgrounds/scenic" />
```

Comment:
If the image is missing, we can have some overlap, see from the AS preview with long text:
<img width="387" alt="image" src="https://user-images.githubusercontent.com/3940906/144286071-d2990628-4cf5-4d22-9e98-df9e40d5e06e.png">
Maybe we can try to avoid that by adding some constraints, even if this is not a blocker

Reply:
will check with design :+1:

Reply:
the title width should be reduced to avoid overlapping https://github.com/vector-im/element-android/pull/4610/commits/ed17b635a9788f552ad028d1494ab03a8cf6a468
![Screenshot_20211202_100941](https://user-images.githubusercontent.com/1848238/144401553-94befcb1-ec65-4fc0-a95f-720ca27277ee.png)